contract-tests/src/bin/sse-test-api/main.rs

```diff
@@ -39,17 +39,24 @@ struct Config {
 enum EventType {
     #[serde(rename = "event")]
     Event { event: Event },
+    #[serde(rename = "comment")]
```

Comment:
I think we can replace these renames with a single rename_all = "camelCase" on the entire enum.